pkg/resources/create.go

```diff
@@ -99,6 +101,9 @@ func Create(logger *zap.SugaredLogger, rt json.RawMessage, triggerName, eventID,
 	logger.Infof("For event ID %q creating resource %v", eventID, gvr)
 
 	if _, err := dc.Resource(gvr).Namespace(namespace).Create(data, metav1.CreateOptions{}); err != nil {
+		if kerrors.IsUnauthorized(err) || kerrors.IsForbidden(err) {
+			return err
```

Comment:
Nice!

Reply:
by the way @dibyom IMO we need this change here regardless of whether we add finer grained auth at the trigger level

thoughts?

Reply:
I totally agree!